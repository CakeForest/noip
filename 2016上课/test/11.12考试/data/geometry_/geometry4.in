10
0 44 80 36 72 78 18 2 68 68 
4 75 6 24 48 80 24 70 14 64 
10
27 44
60 82
94 14
57 0
4 36
50 53
5 20
0 51
0 56
72 6
