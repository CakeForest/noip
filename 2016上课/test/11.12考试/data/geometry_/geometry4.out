8
10
10
5
5
10
3
6
6
9
